resources/lib/scrobbler.py

```diff
@@ -126,14 +126,14 @@ def playbackStarted(self, data):
             elif utilities.isEpisode(self.curVideo['type']):
                 if 'id' in self.curVideo:
                     episodeDetailsKodi = kodiUtilities.getEpisodeDetailsFromKodi(self.curVideo['id'], ['showtitle', 'season', 'episode', 'tvshowid', 'uniqueid', 'file', 'playcount'])
-                    tvdb = episodeDetailsKodi['imdbnumber']
+                    ids = utilities.parseIdToTraktIds(episodeDetailsKodi['imdbnumber'], self.curVideo['type'])[0]
```

Comment:
can you make it `id`? and yes the `parseIdToTraktIds` is also wrongly named, just ignore it for now :)